Add linux install guide (#10)
osuTitanic/wiki: wiki/Installation/en.md

<img src="img/Link-Finished.png">

## Linux Installation

### Step 1: Prerequisites

**Install git**

Ubuntu/Debian: `sudo apt install -y git`                                                                                                                   
Arch Linux: `sudo pacman -Sy --needed  --noconfirm git`                                                                                                    
Fedora: `sudo dnf install -y git`

**Install Lutris**

> Ubuntu/Debian: https://github.com/lutris/lutris/releases // Get the .deb package from here!                                                                
> Arch Linux: `sudo pacman -S lutris`                                                                                                                        
> Fedora: `sudo dnf install lutris`
> 
> Flatpak: `flatpak install flathub net.lutris.Lutris`

Reviewer: ```suggestion
Ubuntu/Debian: https://github.com/lutris/lutris/releases (Get the .deb package from here)
Arch Linux: `sudo pacman -S lutris`                                                                                                                        
Fedora: `sudo dnf install lutris`
Flatpak: `flatpak install flathub net.lutris.Lutris`
```

Looks better in my opinion.

**Install needed drivers**

You can refer to this site
https://github.com/lutris/docs/blob/master/InstallingDrivers.md

**Pipewire**

Pipewire performs much better on osu! than Pulseaudio, you can install it by following:
https://github.com/NelloKudo/osu-winello/wiki/Installing-PipeWire

> ### Step 2: Installation of osu-winello osu! stable client (in order to get his tweaks)

Reviewer: ```suggestion
### Step 2: Installation of osu-winello
```

This should be enough for a headline

Turn off DXVK and VKD3D
![image](https://github.com/user-attachments/assets/e6e2fb21-47a6-49f4-b038-5bf6a667696c)

Make sure ESync and FSync are enabled 
Disable AMD FSR
Disable BattlEye Anticheat
Disable Easy Anticheat

![image](https://github.com/user-attachments/assets/b3b4e8ff-7ef5-40a2-bb26-b1106159eaea)

In System options,
Turn on advanced then click enable "disable Desktop effects"
![image](https://github.com/user-attachments/assets/372564d8-9977-433b-8cb2-663f1c0b8a1a)

Enable Feral Gamemode
![image](https://github.com/user-attachments/assets/d88718f4-580a-41eb-88dd-dc7f74fdc9d4)

Scroll down until you see Environment Variables
Click add
On the left side put WINE_BLOCK_GET_VERSION
On the right side put 1

This is what it should look like.
![image](https://github.com/user-attachments/assets/f5da68e2-2395-44f0-ab23-474798cc8287)

Now from here you can launch osu! and enjoy

**NOTE:** Be sure to set the offset to around `-40` to `-25 ms` offset. Then the audio will be like Windows!

**Note for Wayland users**

You may have to apply the tweaks listed here
https://github.com/NelloKudo/osu-winello/wiki/Wayland-support

> If you have any issues do not be afraid to ask in the [Titanic discord](https://discord.gg/qupv72e7YH) or in the [website forums](https://osu.titanic.sh/forum/).

Reviewer: ```suggestion
If you have any issues do not be afraid to ask in the [Titanic Discord](https://discord.gg/qupv72e7YH) or in the [support forum](https://osu.titanic.sh/forum/7).
```
